test/EVMHost.cpp

```diff
@@ -249,13 +249,33 @@ evmc::result EVMHost::call(evmc_message const& _message) noexcept
 
 	if (message.kind == EVMC_CREATE)
 	{
-		// TODO this is not the right formula
 		// TODO is the nonce incremented on failure, too?
+		sender.nonce++;
+
+		auto encodeRlpInteger = [](int value) -> bytes {
+			if (value == 0) {
+				return bytes(1, 128);
```

Comment:
Are you sure about this?

```
>>> cast --to-rlp '["0xf467baad569be981fb1a781f640de36059cd774a", "00"]'
0xd694f467baad569be981fb1a781f640de36059cd774a00
```

Seems to be mapping to 0?

Reply:
This is right. Usually when encoding integers the leading zero bytes are eliminated before byte-string encoding. In case of 0 you end up with empty bytes `""` encoded as `0x80`.

Reply:
Okay, that adds up
```
>>> cast --to-rlp '["0xf467baad569be981fb1a781f640de36059cd774a", ""]'
0xd694f467baad569be981fb1a781f640de36059cd774a80
```